src/main/java/org/jitsi/jigasi/transcription/GoogleCloudTranscriptionService.java

```diff
@@ -179,12 +179,52 @@
      */
     private final static int STREAMING_SESSION_TIMEOUT_MS = 2000;
 
+    /**
+     * Property name to determine whether to use the Google Speech API's
+     * video model
+     */
+    private final static String P_NAME_USE_VIDEO_MODEL
+        = "org.jitsi.jigasi.transcription.USE_VIDEO_MODEL";
+
+    /**
+     * The default value for the property USE_VIDEO_MODEL
+     */
+    private final static boolean DEFAULT_VALUE_USE_VIDEO_MODEL = false;
+
+    /**
+     * Check whether the given string contains a supported language tag
+     *
+     * @param tag the language tag
+     * @throws UnsupportedOperationException when the google cloud API does not
+     * support the given language
+     */
+    private static void validateLanguageTag(String tag)
+        throws UnsupportedOperationException
```

Comment:
I moved this up because I made some methods which were above non-static, but want to preserve static methods being at top of file.